api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2Spec.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"replicas", "connectCluster", "config", "image",
+        "livenessProbe", "readinessProbe", "jvmOptions",
+        "affinity", "tolerations", "logging", "metrics", "tracing", "template", "clusters", "mirrors"})
+@EqualsAndHashCode(callSuper = true, doNotUseGetters = true)
+public class KafkaMirrorMaker2Spec extends KafkaConnectSpec {
+
+    private static final long serialVersionUID = 1L;
+    
+    private List<KafkaMirrorMaker2ClusterSpec> clusters;
+    private String connectCluster;
+    private List<KafkaMirrorMaker2MirrorSpec> mirrors;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("Kafka clusters for mirroring.")
+    public List<KafkaMirrorMaker2ClusterSpec> getClusters() {
+        return clusters;
+    }
+
+    public void setClusters(List<KafkaMirrorMaker2ClusterSpec> clusters) {
+        this.clusters = clusters;
+    }
+
+    @Description("The cluster alias used for Kafka Connect.")
```

Comment:
Maybe add "This should be the name of one of the aliases given in..."?

Reply:
Done. I have followed the wording used elsewhere: 

    "The cluster alias used for Kafka Connect. The alias must match a cluster in the 
    list at spec.clusters."